pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -715,6 +718,9 @@ func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
 
 // DaemonSets should not unschedule a daemonset pod from a node with insufficient free resource
 func TestInsufficientCapacityNodeDaemonDoesNotUnscheduleRunningPod(t *testing.T) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods) {
```

Comment:
Does this test fail when the feature is enabled? I would keep this test to ensure that DS controller does not react to out of resource nodes.

Reply:
There are create and delete events mismatch that would happen after enabling this feature. I think, we can modify them to reflect the new changes just like I did for other tests instead of skipping. I will make that change.